src/elf_parser/mod.rs

```diff
@@ -132,7 +132,6 @@ impl<'a> Elf64<'a> {
             slice_from_bytes::<Elf64Shdr>(elf_bytes, section_header_table_range.clone())?;
         section_header_table
             .get(0)
-            .map(|section_header| section_header.sh_type == SHT_NULL)
```

Comment:
shouldn't we fix this by replacing `map` with [`filter`](https://doc.rust-lang.org/std/option/enum.Option.html#method.filter)?

Reply:
The new ELF parser is not deployed yet, so we can still change the behavior.